Platform/OpenCanopy/BitmapFont.c

```diff
@@ -618,11 +618,13 @@ BlendMem (
   UINTN  Index;
 
   for (Index = 0; Index < PixelCount; ++Index) {
-    //
-    // We assume that the font is generated by dpFontBaker
-    // and has only gray channel, which should be interpreted as alpha.
-    //
-    GuiBlendPixel(Dst, Color, AlphaSrc->Red);
+    if (AlphaSrc->Red != 0) {
```

Comment:
Are you sure this is a speedup? Conditionals noticeable slowdown code in general.

Reply:
This is a tradeoff, this is code that is called once per label on initialisation, and it saves branches in the main draw loop in return.

Reply:
Isn't it per-pixel?

Reply:
@vit9696 Yes, the point is it slows down only initialisation (and only slightly, as labels are small), while the other paths, that are improved, are in the main drawing loop.

Reply:
@vit9696 Hmm, actually I realised the code works for 0 opacity (I thought there was a division by 0 with the new code), do you want me to drop the branch?

Reply:
I'd rather profile.

Reply:
I don't think I really have time for that, could leave it as-is and add a TODO